api_app/playbooks_manager/views.py

```diff
@@ -20,7 +20,11 @@
 logger = logging.getLogger(__name__)
 
 
-class PlaybookConfigViewSet(AbstractConfigViewSet, mixins.CreateModelMixin):
+class PlaybookConfigViewSet(
+    AbstractConfigViewSet,
+    mixins.CreateModelMixin,
+    mixins.DestroyModelMixin,
+):
```

Comment:
If you do not change the permissions here, every user can call the API directly and delete every playbook.